packages/model-viewer/src/features/animation.ts

```diff
@@ -101,6 +101,16 @@ export const AnimationMixin = <T extends Constructor<ModelViewerElementBase>>(
       this[$needsRender]();
     }
 
+    get currentTimeScale(): number {
+      return this[$scene].animationTimeScale;
+    }
+
+    set currentTimeScale(value: number) {
+      this[$scene].animationTimeScale = value;
+      this[$renderer].threeRenderer.shadowMap.needsUpdate = true;
+      this[$needsRender]();
```

Comment:
Are these renderer updates necessary? If the animation is playing, these are being called anyway, and if not, then changing the time scale shouldn't affect the  render, right?

Reply:
fix it